wordpress-mobile/WordPress-Android
# Add analytics to image editor

Fixes #10984

This PR adds given analytics tracks events to track usage.

To test:
- Ensure `Collect Information` option is enabled in user's App's Settings -> Privacy settings
- Launch image editor by clicking edit option for an image in the Gutenberg editor (need to enable edit option) 
- Check that the console contains the following log:
    - `🔵 Tracked: media_editor_shown`
- Click Done (check) button on Crop screen
- Check that the console contains the following log:
   - `🔵 Tracked: media_editor_used, Properties: {"actions":"crop"}`

P.S. `uCrop` doesn't send any data in crop result to differentiate between `crop`, `rotate`, `scale` actions. As per internal conversation, tracking just `crop` action on done button click is sufficient based on iOS tracks analysis to avoid any hack.

PR submission checklist:
